routing/result_interpretation_test.go

```diff
@@ -692,6 +692,25 @@ var resultTestCases = []resultTestCase{
 			finalFailureReason: &reasonError,
 		},
 	},
+	// Test a multi-hop blinded route and that in a success case the amounts
+	// for the blinded route part are correctly set to the receiver amount.
+	{
+		name:    "blinded multi-hop success",
+		route:   blindedMultiToIntroduction,
```

Comment:
We should also update `blindedMultiToIntroduction` and `blindedMultiHop` to be more realistic, to set the correct amounts to zero (I think including the intro node hop until the penultimate hop)

Reply:
correct including the introduction point.